packages/relay/src/lib/eth.ts

```diff
@@ -493,7 +493,15 @@ export class EthImpl implements Eth {
           const accountCacheKey = `${constants.CACHE_KEY.ACCOUNT}_${transaction.to}`;
           let toAccount: object | null = this.cache.get(accountCacheKey);
           if (!toAccount) {
-            toAccount = await this.mirrorNodeClient.getAccount(transaction.to, requestId);
+            try {
```

Comment:
Extract to a shared method doing the logic and returning the account, null or throwing an error